aws/backbone/route_tables.tf

```diff
@@ -2,14 +2,30 @@
 # Public Route Table
 #
 
+locals {
+  public_name = {
```

Comment:
Given this is a map, I would have call it `public_names` (same for `private_name`). Thoughts?

Reply:
There is only one public / private name (wouldn't names suggest a list?)

Reply:
Ok! (yes it would)